Story widget

tRy-HaRd-tgn opened the pull request.

tRy-HaRd-tgn committed:
fix: добавил типы на инпуты, инпут города заменил
на тот, который сделал тема

tRy-HaRd-tgn committed:
feat: скачал библиотеку, написал функцию нажатия
которая выводит истории

Hubulbib commented on src/pages/auth-2/index.jsx:
```diff
@@ -14,26 +15,38 @@ export const Auth2 = () => {
+    types: ["email", "password", "text"],
   };
-
+  const townsList = [
```
`townList` можно вынести как константу в model

Hubulbib reviewed (approved):
Необходимо добавить кнопку выхода из сториса.
Сделать анимацию появления.

tRy-HaRd-tgn committed:
feat: добавил see more и кнопку выхода из истории

tRy-HaRd-tgn committed:
fix: поправил в своих input width(100vh-45px)
это не особо подходит под логику того кода
который написал собар, нпуты растягивались
fix: поправил также select артема, т.к он исполь-
зует ту же логику
fix: исправил расположение кнопки и изменил над-
пись кнопки

tRy-HaRd-tgn committed:
fix: исправил страничку profile, в связи с новыми
стилями для input

tRy-HaRd-tgn committed:
fix: изменил defaultInterval с 1500 на 2000
fix: добавил padding 15px
